backend/.env.example

```
@@ -1,5 +1,5 @@
 CORS_ORIGINS=["http://localhost:5173"]
 CORS_ORIGIN_REGEX=
 OPEN_METEO_BASE_URL=https://api.open-meteo.com/v1
-HTTP_TIMEOUT_SECONDS=10
+HTTP_TIMEOUT_SECONDS=3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HTTP_TIMEOUT_SECONDS key should go before the OPEN_METEO_BASE_URL key

(UnorderedKey)